diff --git a/src/Plugin/resource/Antidotes__1_0.php b/src/Plugin/resource/Antidotes__1_0.php
new file mode 100755
index 0000000..c4cfae0
--- /dev/null
+++ b/src/Plugin/resource/Antidotes__1_0.php
@@ -0,0 +1,171 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\restful\Plugin\resource\Antidotes__1_0.
+ */
+
+namespace Drupal\restful\Plugin\resource;
+
+use Drupal\restful\Http\RequestInterface;
+use Drupal\restful\Plugin\resource\ResourceEntity;
+use Drupal\restful\Plugin\resource\ResourceInterface;
+use Drupal\restful\Plugin\resource\ResourceNode;
+
+/**
+ * Class Articles
+ * @package Drupal\restful\Plugin\resource
+ *
+ * @Resource(
+ *   name = "antidotes:1.0",
+ *   resource = "antidotes",
+ *   label = "Antidotes",
+ *   description = "Export the Antidote entity with cookie authentication.",
+ *   authenticationTypes = TRUE,
+ *   authenticationOptional = TRUE,
+ *   dataProvider = {
+ *     "entityType": "node",
+ *     "bundles": {
+ *       "antidote"
+ *     },
+ *   },
+ *   majorVersion = 1,
+ *   minorVersion = 0
+ * )
+ */
+class Antidotes__1_0 extends ResourceEntity implements ResourceInterface {
+  /**
+   * {@inheritdoc}
+   */
+  protected function publicFields() {
+    $public_fields = parent::publicFields();
+
+    // Last updated
+    $public_fields['last_updated'] = array(
+      'property' => 'changed',
+    );
+
+    // Language
+    $public_fields['language'] = array(
+      'property' => 'language',
+    );
+
+    // Indications
+    $public_fields['indications'] = array(
+      'property' => 'field_indications',
+    );
+
+    // Posologie
+    $public_fields['posologie'] = array(
+      'property' => 'field_posologie',
+    );
+
+    // Points clés
+    $public_fields['points_cles'] = array(
+      'property' => 'field_points_cles',
+    );
+
+    // Synonymes et autres appellations 
+    $public_fields['synonymes_et_autres_appell'] = array(
+      'property' => 'field_synonymes_et_autres_appell',
+    );
+
+    // Effets indésirables
+    $public_fields['effets_indesirables'] = array(
+      'property' => 'field_effets_indesirables',
+    );
+
+    // Monitoring
+    $public_fields['monitoring'] = array(
+      'property' => 'field_monitoring',
+    );
+
+    // Fin du traitement
+    $public_fields['fin_du_traitement'] = array(
+      'property' => 'field_fin_du_traitement',
+    );
+
+    // Particularités reliées à l’administration
+    $public_fields['particularites_reliees'] = array(
+      'property' => 'field_particularites_reliees',
+    );
+
+    // Produits disponibles
+    $public_fields['produits_disponibles'] = array(
+      'property' => 'field_produits_disponibles',
+    );
+
+    // Stockage recommandé
+    $public_fields['stockage_recommande'] = array(
+      'property' => 'field_stockage_recommande',
+    );
+
+    // Calculatrice
+    $public_fields['calculatrice'] = array(
+      'property' => 'field_calculatrice',
+    );
+
+    // Dose maximale
+    $public_fields['dose_maximale'] = array(
+      'property' => 'field_dose_maximale',
+    );
+
+    // Dose par poids
+    $public_fields['dose_par_poids'] = array(
+      'property' => 'field_dose_par_poids',
+    );
+
+    // Dose maximale enfant
+    $public_fields['dose_maximale_enfant'] = array(
+      'property' => 'field_dose_maximale_enfant',
+    );
+
+    // Dose par poids enfant
+    $public_fields['dose_par_poids_enfant'] = array(
+      'property' => 'field_dose_par_poids_enfant',
+    );
+
+    // Références
+    $public_fields['references'] = array(
+      'property' => 'field_references',
+    );
+
+    // Dose pédiatrique
+    $public_fields['field_pediatrique'] = array(
+      'property' => 'field_pediatrique',
+    );
+
+    // Dose adulte
+    $public_fields['field_adulte'] = array(
+      'property' => 'field_adulte',
+    );
+
+    // Insuffisance rénale
+    $public_fields['field_renale'] = array(
+      'property' => 'field_renale',
+    );
+
+     // Insuffisance hépatique
+    $public_fields['field_hepatique'] = array(
+      'property' => 'field_hepatique',
+    );
+
+    // Patient hémodialysé
+    $public_fields['field_hemodialyse'] = array(
+      'property' => 'field_hemodialyse',
+    );
+
+    // Grossesse
+    $public_fields['field_grossesse'] = array(
+      'property' => 'field_grossesse',
+    );
+
+    // Patient obèse ou avec un surplus de poids
+    $public_fields['field_obese'] = array(
+      'property' => 'field_obese',
+    );
+
+    return $public_fields;
+  }
+
+}
diff --git a/src/Plugin/resource/Centres__1_0.php b/src/Plugin/resource/Centres__1_0.php
new file mode 100755
index 0000000..3e798a5
--- /dev/null
+++ b/src/Plugin/resource/Centres__1_0.php
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\restful\Plugin\resource\Centres__1_0.
+ */
+
+namespace Drupal\restful\Plugin\resource;
+
+use Drupal\restful\Http\RequestInterface;
+use Drupal\restful\Plugin\resource\ResourceEntity;
+use Drupal\restful\Plugin\resource\ResourceInterface;
+use Drupal\restful\Plugin\resource\ResourceNode;
+
+/**
+ * Class Articles
+ * @package Drupal\restful\Plugin\resource
+ *
+ * @Resource(
+ *   name = "centres:1.0",
+ *   resource = "centres",
+ *   label = "Centres",
+ *   description = "Export the Centre entity with cookie authentication.",
+ *   authenticationTypes = TRUE,
+ *   authenticationOptional = TRUE,
+ *   dataProvider = {
+ *     "entityType": "node",
+ *     "bundles": {
+ *       "centre_antipoison"
+ *     },
+ *   },
+ *   majorVersion = 1,
+ *   minorVersion = 0
+ * )
+ */
+class Centres__1_0 extends ResourceEntity implements ResourceInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function publicFields() {
+    $public_fields = parent::publicFields();
+
+    // Téléphone
+    $public_fields['telephone'] = array(
+      'property' => 'field_telephone',
+    );
+
+    // Site web
+    $public_fields['site_web'] = array(
+      'property' => 'field_site_web',
+    );
+
+    // Image URL
+    $public_fields['image_url'] = array(
+      'property' => 'field_image_url',
+    );
+
+    // Ordre index
+    $public_fields['ordre_index'] = array(
+      'property' => 'field_ordre_index',
+    );
+
+    return $public_fields;
+  }
+
+}
diff --git a/src/Plugin/resource/DataProvider/DataProvider.php b/src/Plugin/resource/DataProvider/DataProvider.php
old mode 100644
new mode 100755
index 1d2a8ac..e075cad
--- a/src/Plugin/resource/DataProvider/DataProvider.php
+++ b/src/Plugin/resource/DataProvider/DataProvider.php
@@ -39,7 +39,7 @@ abstract class DataProvider implements DataProviderInterface {
    *
    * @var int
    */
-  protected $range = 50;
+  protected $range = 500;
 
   /**
    * The account authenticated from the request for entity access checks.
diff --git a/src/Plugin/resource/Social__1_0.php b/src/Plugin/resource/Social__1_0.php
new file mode 100755
index 0000000..f0c6c68
--- /dev/null
+++ b/src/Plugin/resource/Social__1_0.php
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\restful\Plugin\resource\Social__1_0.
+ */
+
+namespace Drupal\restful\Plugin\resource;
+
+use Drupal\restful\Http\RequestInterface;
+use Drupal\restful\Plugin\resource\ResourceEntity;
+use Drupal\restful\Plugin\resource\ResourceInterface;
+use Drupal\restful\Plugin\resource\ResourceNode;
+
+/**
+ * Class Articles
+ * @package Drupal\restful\Plugin\resource
+ *
+ * @Resource(
+ *   name = "social:1.0",
+ *   resource = "social",
+ *   label = "Social",
+ *   description = "Export the Social entity with cookie authentication.",
+ *   authenticationTypes = TRUE,
+ *   authenticationOptional = TRUE,
+ *   dataProvider = {
+ *     "entityType": "node",
+ *     "bundles": {
+ *       "social"
+ *     },
+ *   },
+ *   majorVersion = 1,
+ *   minorVersion = 0
+ * )
+ */
+class Social__1_0 extends ResourceEntity implements ResourceInterface {
+  /**
+   * {@inheritdoc}
+   */
+  protected function publicFields() {
+    $public_fields = parent::publicFields();
+
+    // Facebook
+    $public_fields['page_facebook'] = array(
+      'property' => 'field_page_facebook',
+    );
+
+    // Twitter
+    $public_fields['page_twitter'] = array(
+      'property' => 'field_page_twitter',
+    );
+
+    // Courriel
+    $public_fields['page_courriel'] = array(
+      'property' => 'field_page_courriel',
+    );
+
+    return $public_fields;
+  }
+
+}
